### DISCUSSION TITLE
Drop support for Scala 2.12

### DISCUSSION THREAD

**Discussion by Qi77Qi** [broadinstitute/workbench-libs#441] [General]
Services depends on workbench-libs that have migrated to Scala 2.13

- [x] Leonardo
- [x] Cromwell
- Seems like WIP?
- [x] Orchestration
- broadinstitute/firecloud-orchestration#825
- [x] Rawls (note rawls-model is already cross publishing to 2.13)
- Depends on Cromwell
- [x] Agora
- [x] Sam
~~Need to bump scalatest version still~~
- ~~https://github.com/broadinstitute/sam/pull/491~~

Related PR
- https://github.com/broadinstitute/workbench-libs/pull/461

**Comment by MatthewBemis** [comment]
Orchestration is almost there. I have a draft PR for it here: https://github.com/broadinstitute/firecloud-orchestration/pull/825

Outstanding issues are migrating the automation package to 2.13 and fixing some unit tests that broke because of collection ordering changes. It's not high priority work at the moment but I have been chipping away at it during spare time.
